strings/check_if_string_can_be_converted_to_palindrome.py

```diff
@@ -0,0 +1,60 @@
+# Created by susmith98
+
+# Problem Description:
+# Check if characters of the given string can be rearranged to form a palindrome.
+
+def check_if_string_can_be_rearranged_as_palindrome(input_str: str = "",) -> bool:
+    """
+    A Palindrome is a String that reads the same forward as it does backwards.
+    Examples of Palindromes mom, dad, malayalam
+    >>> check_if_string_can_be_rearranged_as_palindrome("Momo")
+    True
+    >>> check_if_string_can_be_rearranged_as_palindrome("Mother")
+    False
+    >>> check_if_string_can_be_rearranged_as_palindrome("Father")
+    False
+    """
+    if len(input_str) == 0:
```

Comment:
```suggestion
    if len(input_str) < 2:
```

Reply:
I'll make the requested Changes @cclauss 👍 . Is it okay if I leave the existing code (will comment it out)? Just thought it would be readily understandable for beginners.

Reply:
Sure.  Or you can create two functions with the same signature but different names and add a [timeit]() benchmark to prove which one is faster.

Reply:
Done. Made the changes as specified 👍